doc/guides/west/install.rst

```diff
@@ -0,0 +1,86 @@
+.. _west-install:
+
+Installing west
+###############
+
+West is written in Python 3 and distributed through `PyPI`_.
+Use :file:`pip3` to install or update west:
+
+On GNU/Linux::
+
+  pip3 install --user -U west
+
+On Windows and macOS::
+
+  pip3 install -U west
+
+.. note::
+   See :ref:`gs_python_deps` for additional clarfication on using the
+   ``--user`` switch.
+
+The following are the typical locations where the west files will be
+installed if you followed the instructions above:
+
+* GNU/Linux:
+
+  * Executable: :file:`~/.local/bin/west`
+  * Package: :file:`~/.local/lib/python3.<x>/site-packages/west/`
+
+* macOS:
+
+  * Executable: :file:`/usr/local/bin/west`
+  * Package: :file:`/usr/local/lib/python3.<x>/site-packages/west/`
+
+* Windows:
+
+  * Executable: :file:`c:\Python3<x>\Scripts\west.exe`
+  * Package: :file:`C:\Python3<x>\Lib\site-packages\west`
+
+Once west is installed you can use it to :ref:`clone the Zephyr repositories
+<clone-zephyr>`.
+
+.. _west-shell-completion:
+
+Enabling shell completion
```

Comment:
I disagree here. I think this belongs in the installation guide because it's the sort of thing that users associate with installing. Also, I want to add an additional comment about remembering to update the completion file every time the user updates west with `pip`, and it would be nice to keep those 2 concepts together.

Reply:
Then I think you can remove it from the miscellaneous file.

It's not the case that that's the catch-all location for documenting non build/flash/debug commands. The sign command has its own page, for example.

Reply:
Done